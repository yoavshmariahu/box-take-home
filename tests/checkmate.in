k b1
K e5
R a5
s e3
g d3

[B R P]
[p b s g]

move d3 e4
