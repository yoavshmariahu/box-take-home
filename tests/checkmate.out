lower player action: move d3 e4
5 | R|__|__|__| K|
4 |__|__|__|__| g|
3 |__|__|__|__| s|
2 |__|__|__|__|__|
1 |__| k|__|__|__|
    a  b  c  d  e

Captures UPPER: B R P
Captures lower: p b s g

lower player wins.  Checkmate.
